Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook

```diff
@@ -151,152 +152,720 @@
et totalOperationsByTypeSeries = operationData\r\n| make-series TotalOperationsByTypeSeries = sum(Rate) default = 0 on TimeGenerated from ago(21600s) to now() step 10m by OperationType\r\n| project-away TimeGenerated;\r\nlet errorsData = data\r\n| where Name in ('kubelet_docker_operations_errors','kubelet_runtime_operations_errors');\r\nlet totalErrorsByType = errorsData\r\n| summarize Rate = sum(Rate) by OperationType\r\n| project OperationType, TotalErrors = Rate;\r\nlet totalErrorsByTypeSeries = errorsData\r\n| make-series TotalErrorsByTypeSeries = sum(Rate) default = 0 on TimeGenerated from ago(21600s) to now() step 10m by OperationType\r\n| project-away TimeGenerated;\r\nlet seriesLength = toscalar(   totalOperationsByTypeSeries\r\n| extend ArrayLength = array_length(TotalOperationsByTypeSeries)\r\n| summarize Array_Length = max(ArrayLength)  );\r\ntotalOperationsByType\r\n| join kind = leftouter (\r\n    totalErrorsByType\r\n) on OperationType\r\n| project-away OperationType1\r\n| extend TotalErrors = iif(isempty(TotalErrors), 0.0, TotalErrors)\r\n| join kind = leftouter (\r\n    totalErrorsByTypeSeries\r\n) on OperationType\r\n| project-away OperationType1\r\n| extend SeriesOfEqualLength = range(1, seriesLength, 1)\r\n| extend SeriesOfZeroes = series_subtract(SeriesOfEqualLength, SeriesOfEqualLength)\r\n| extend SeriesOfOneHundo = series_multiply(series_divide(SeriesOfEqualLength, SeriesOfEqualLength), 100)\r\n| extend TotalErrorsByTypeSeries = iif(isempty(TotalErrorsByTypeSeries), SeriesOfZeroes, TotalErrorsByTypeSeries)\r\n| join kind=leftouter (\r\n    totalOperationsByTypeSeries\r\n) on OperationType\r\n| project-away OperationType1\r\n| extend TotalSuccessfulOperationsByTypeSeries = series_subtract(TotalOperationsByTypeSeries, TotalErrorsByTypeSeries)\r\n| extend IsNegativeTotalSuccessfulOperationsByTypeSeries = series_less(TotalSuccessfulOperationsByTypeSeries, SeriesOfZeroes)\r\n| extend TotalSuccessfulOperationsByTypeSeries = array_iif(IsNegativeTotalSuccessfulOperationsByTypeSeries, SeriesOfZeroes, TotalSuccessfulOperationsByTypeSeries) \r\n| extend SuccessPercentage = round(iif(TotalOperations == 0, 1.0, iif(TotalErrors > TotalOperations, 0.0, 1 - (TotalErrors / TotalOperations))), 4), SuccessPercentageSeries = series_divide(TotalSuccessfulOperationsByTypeSeries, TotalOperationsByTypeSeries)\r\n| extend SuccessfulOperationsEqualsTotalOperationsSeries = series_equals(TotalSuccessfulOperationsByTypeSeries, TotalOperationsByTypeSeries)\r\n| extend SuccessPercentageSeries = array_iff(SuccessfulOperationsEqualsTotalOperationsSeries, SeriesOfOneHundo, SuccessPercentageSeries)\r\n| project OperationType, TotalOperations, TotalErrors, SuccessPercentage, SuccessPercentageSeries\r\n| order by SuccessPercentage asc, OperationType asc\r\n| project-rename ['Operation Type'] = OperationType, ['Total Operations'] = TotalOperations, ['Total Errors'] = TotalErrors, ['Success %'] = SuccessPercentage, ['Success % Trend'] = SuccessPercentageSeries",
```

Comment:
Here as well . I see my recent changes to kubelet workbook not here.  can you please take latest merged changes to this work book?

Reply:
Fixed